pkg/lib/serverTime.js

```diff
@@ -595,8 +595,8 @@
         <Form isHorizontal>
             <FormGroup fieldId="systime-timezones" label={_("Time zone")}>
                 <Validated errors={errors} error_key="time_zone">
-                    <Select id="systime-timezones" variant={SelectVariant.typeahead}
-                            isOpen={zonesOpen} onToggle={setZonesOpen}
+                    <Select id="systime-timezones" variant="typeahead"
+                            isOpen={zonesOpen} onToggle={(_, isOpen) => setZonesOpen(isOpen)}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18672-20230426-085951-0a013b1d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)